client/src/utils/path-parsers.ts

```diff
@@ -19,5 +19,11 @@ export const isLanding = (pathname: string): boolean => {
   return isEnglishLanding || isI18Landing;
 };
 
+export const isLocationSuperBlock = (
+  location: Window['location'] | undefined
+): boolean => {
+  return /^\/learn\/\w+\/$/.test(location?.pathname ?? '');
```

Comment:
```suggestion
export const isLocationSuperBlock = (location: WindowLocation): boolean => {
  return /^\/learn\/[a-z-]+\/?$/.test(location.pathname ?? '');
```
`\w` doesn't include `-`, unfortunately.  Also I made the last `/` optional, just in case it was trimmed for whatever reason.

It also needs `import type { WindowLocation } from '@reach/router';` at the top of the file, but GitHub won't let me suggest that directly :(

Reply:
That is so odd. I originally had `[\w-]+`, but that change appears to be missing.

Also, I did not think `WindowLocation` necessary, as, after looking at `reach/router` it appears they just directly use `Window`. So, better to not import it 🤷 . I am not sure why the commit includes the package.